attestation-service/docker/as-restful/Dockerfile

```diff
@@ -64,6 +64,10 @@ COPY ./attestation-service/config.json /etc/config.json
 
 VOLUME /opt/confidential-containers/attestation-service
 
-CMD ["restful-as", "--socket", "0.0.0.0:8080", "--config-file", "/etc/config.json"]
+ENV ALLOWED_REGIONS="http://localhost"
+
+CMD exec restful-as --socket 0.0.0.0:8080 --config-file /etc/config.json --allowed_region ${ALLOWED_REGIONS}
```

Comment:
I prefer not to change this file, and let users add parameters to container command if they need CORS. wdyt?